### ISSUE TITLE
Lightweight tag &  git.closest.tag.name ?

### ISSUE THREAD

**Issue by jcnoir** [ktoso/maven-git-commit-id-plugin#221]
I use this plugin with lightweight tags.
I configure the pom.xml like this : 

``` xml
<gitDescribe>
<tags>true</tags>
</gitDescribe>
```

In the generated git.properties file :
- git.tags : OK (points to the last lightweight tag)
- git.closest.tag.name : points to the last annotated tag and not the last lightweight tag.

Is there a way to get the  last lightweight tag in the  git.closest.tag.name property ?

Thanks for this nice plugin !

**Comment by TheSnoozer** [comment]
Hi Thanks for reporting your Problem here.
I'll try to setup a small project to reproduce the Problem the have...stay tuned for updates!! :-)

**Comment by TheSnoozer** [comment]
the closest tag and clostest.tag.count will be determined based on properties configured inside the git-describe command:
```xml
<gitDescribe>
<tags>true</tags>
</gitDescribe>
```

will cause that lightweight tags are also included for the closest tags

Also the ``pattern`` used inside the gitDescribe might have an impact on the resulting ``git.closest.tag.name`` and ``git.closest.tag.count``

Feature will be available in {{2.2.4}}
